test/src/unit-capi-serialized_queries.cc

```diff
@@ -58,7 +58,7 @@ using ResultSetType = std::map<std::string, std::any>;
 namespace {
 
 template <class T>
-bool check_result(const T a, const T b, size_t start, size_t end) {
+bool check_result(const T& a, const T& b, size_t start, size_t end) {
```

Comment:
Oh it was that easy to fix. 😂😂 Great work!

Reply:
I had to read the error message like 30 times to figure out what was going on :D